package.json

```diff
@@ -11,21 +11,24 @@
   ],
   "scripts": {
     "analyze": "size-limit --why",
-    "build": "dts build",
-    "lint": "dts lint --config \"./.prettierrc.yml\"",
-    "prepare": "dts build",
+    "build": "tsc",
+    "lint": "eslint --config \"./.eslintrc.json\" \"**/*.{json,ts,yml}\" && prettier --config \"./.prettierrc.json\" --write \"**/*.{json,md,js,ts,yml}\"",
+    "prepare": "tsc",
     "size": "size-limit",
-    "start": "dts watch",
-    "test": "dts test",
-    "coverage": "dts test --coverage"
+    "start": "jest --watch",
```

Comment:
start seems like a bit of a misleading name for a running jest tests

Reply:
that's right! I think I'd just remove this and if there's a need for a running jest tests --watch can be appended to yarn test command

Reply:
removed it